src/tooltip/tooltip.spec.ts

```diff
@@ -60,6 +61,78 @@ describe('ngb-tooltip-window', () => {
     expect(fixture.nativeElement).toHaveCssClass('my-custom-class');
   });
 
+  it('should allow to change custom class to new one', () => {
```

Comment:
This is kinda covered by the previous test in this file...

What you should actually test is that:
- when you change `classThatChanges` → `<div [ngbTooltip]="hi" [tooltipClass]="classThatChanges">`
- it actually changes class on the `NgbTooltipWindow`

Reply:
redacted, thanks!